packages/modules/src/modules/epics/ContributionsEpic.stories.tsx

```diff
@@ -6,6 +6,7 @@ import { from } from '@guardian/src-foundations/mq';
 import { css } from '@emotion/react';
 import { palette } from '@guardian/src-foundations';
 import { EpicProps, SecondaryCtaType, TickerCountType, TickerEndType } from '@sdc/shared/types';
+import { UK_DATA } from '@sdc/server/src/tests/epics/choiceCardsTestData';
```

Comment:
It's nice to save duplication, but I think we don't want to import from `@sdc/server` in `@sdc/modules` and vice versa. It's nice and safe if we just make a hard rule there to prevent mixing client + server code

Reply:
👍🏼